client-v2/README.md

```diff
@@ -18,30 +18,13 @@
 
 Fronts Client V2 looks to rebuild the Fronts tool with modern technologies, develop reusable patterns for content curation and build shareable components for across the tools.
 
-## Setup
+## Dev Setup
 
-To setup the Fronts V2 project:
-
-```bash
-./scripts/setup.sh
-```
-
-View [setup.sh](scripts/setup.sh).
-
-## Developing
-
-To start developing, first get [Janus credentials](https://janus.gutools.co.uk/multi-credentials?&permissionIds=cmsFronts-dev,capi-api-gateway,frontend-dev) then run:
-
-```bash
-./scripts/client-dev.sh
-```
-
-The v2 application will be running here:
-[https://fronts.local.dev-gutools.co.uk/v2](https://fronts.local.dev-gutools.co.uk/v2)
-
-View [client-dev.sh](scripts/client-dev.sh).
-
-If you wish to use git grep to search the code base you can use [fronts-find.sh](scripts/fronts-find.sh)
+1. Get credentials for `composer` and `workflow` from [Janus](https://janus.gutools.co.uk/multi-credentials?&permissionIds=cmsFronts-dev,capi-api-gateway,frontend-dev).
```

Comment:
And CMS Fronts perms

Reply:
done

Reply:
These hard-coded links will have their expiry time be 1-hour off during BST because they don't include the timezone parameter.

If people are wanting to link directly to Janus permission sets then we should look at adding an interstitial page that redirects to credentials having asked the user's browser for its timezone.

I know some people have bookmarked Janus credentials which would have the same problem, so this probably is worth looking at.

Nothing fundamentally wrong with this PR but the expiry time might be mildly annoying as-is. You could use a BST link for now to address the common use-case for the majority of today's developers, and I'll try and make sure we address the issue in Janus ahead of the hour change at the end of October.